docs/release-notes/python-sdk.rst

```diff
@@ -0,0 +1,48 @@
+:orphan:
+
+**New Features**
+
+-  Python SDK: various new features and enhancements. A few highlights are listed below.
+      -  Add support for downloading a zipped archive of experiment code
+         (:meth:`Experiment.download_code
+         <determined.experimental.client.Experiment.download_code>`).
+
+      -  Add support for :class:`~determined.experimental.client.Project` and
+         :class:`~determined.experimental.client.Workspace` as SDK objects.
+
+      -  Surface more attributes to resource classes, including ``hparams`` and ``summary_metrics``
+         for :class:`~determined.experimental.client.Trial`.
+
+      -  Add support for filtering trial logs by timestamp and a query string using
+         :meth:`Trial.iter_logs <determined.experimental.client.Trial.iter_logs>`.
+
+      -  All resource objects now cache attributes by default and provide a ``.reload()`` method
```

Comment:
It was already sort of true that resource objects "cached attributes." I think I'd prefer this just talk about reload. Like
```
- All resource objects now have a ``.reload()`` method that refreshes the resource's attributes from <don't know what the right word is, here>. Previously, attributes were most easily refreshed by creating an entirely new object.
```